HPR-1172: Add `hcp_packer_channel_assignment` Resource (#531)
hashicorp/terraform-provider-hcp: docs/guides/packer-channel-management.md

---
subcategory: ""
page_title: "Advanced Packer Channel Management - HCP Provider"
description: |-
    A guide to integreting HCP Packer resources and data sources for more advanced channel management.
---

# Advanced Packer Channel Management

> By integrating multiple HCP Packer resources and data sources, you can perform more advanced channel management tasks.

Reviewer: Slight reword to clarify the intent of the guide a bit. 
```suggestion
You can integrate multiple HCP Packer resources and data sources to perform advanced channel management tasks.
```
